lms/djangoapps/verify_student/management/commands/send_verification_expiry_email.py

```diff
@@ -0,0 +1,166 @@
+            'help_center_link': settings.ID_VERIFICATION_SUPPORT_LINK
+        }
+        self.email_context = {
+            'subject': subject,
+            'template': 'emails/verification_expiry_email.txt',
+        }
+
+    def handle(self, *args, **options):
+        """
+        Handler for the command
+        """
+        resend_days = options['resend_days']
+        batch_size = options['batch_size']
+        sleep_time = options['sleep_time']
+
+        date_resend_days_ago = datetime.now(UTC) - timedelta(days=resend_days)
+
+        try:
+            max_user_id = self.sspv.last().user_id
+            batch_start = self.sspv.first().user_id
+            batch_stop = batch_start + batch_size
+
+        except AttributeError:
+            logger.info("AttributeError: No approved expired entries found in SoftwareSecurePhotoVerification")
+            return
+
+        while batch_start <= max_user_id:
+            batch_queryset = self.sspv.filter(user_id__gte=batch_start, user_id__lt=batch_stop)
+            users = batch_queryset.values('user_id').distinct()
+
+            for user in users:
+                verification = self.find_recent_verification(user['user_id'])
+                if not verification.expiry_email_date or verification.expiry_email_date < date_resend_days_ago:
+                    user = User.objects.get(id=verification.user_id)
+                    self.verification_expiry_email_vars['full_name'] = user.profile.name
+                    self.email_context['email_vars'] = self.verification_expiry_email_vars
+                    self.email_context['email'] = user.email
+
+                    verification_qs = self.sspv.filter(pk=verification.pk)
+                    send_verification_expiry_email.delay(self.email_context, verification_qs)
+
+            if batch_stop < max_user_id:
+                time.sleep(sleep_time)
+
+            batch_start = batch_stop
+            batch_stop += batch_size
+
+    def find_recent_verification(self, user_id):
+        """
+        Find the most recent expired verification for user.
+        """
+        return self.sspv.filter(user_id=user_id).latest('updated_at')
+
+
+@task(routing_key=ACE_ROUTING_KEY)
+def send_verification_expiry_email(context, verification):
+    """
+    Spins a task to send verification expiry email to the learner
+    If the email is successfully sent change the expiry_email_date to reflect when the
+    email was sent
+    """
+    subject = context.get('subject')
+    message = render_to_string(context.get('template'), context.get('email_vars'))
+    from_addr = configuration_helpers.get_value(
+        'email_from_address',
+        settings.DEFAULT_FROM_EMAIL
+    )
+    dest_addr = context.get('email')
+
+    try:
+        send_mail(
+            subject,
+            message,
+            from_addr,
+            [dest_addr],
+            fail_silently=False
+        )
```

Comment:
Shouldn't we be using the `edx_ace` library for this?

Reply:
The reasoning for not using `edx_ace` is given [here](https://openedx.atlassian.net/browse/LEARNER-1489?focusedCommentId=321291&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-321291). 
In short it makes three network calls for each user(given our scenario that we need to know if email was sent)